docs/benchmark.rst

```diff
@@ -0,0 +1,43 @@
+Benchmarks
+==========
+
+The results are stored in ``flamby/results`` in corresponding subfolders
+``results_benchmark_fed_dataset`` for each dataset. These results can be
+plotted using:
+
+::
+
+    python plot_results.py
+
+which produces the plot found at the end of the main article.
+
+In order to re-run each of the benchmark on your machine, first download
```

Comment:
do you think reminding that different set of installations might be needed might be useful?

Reply:
Good point ! Lemme change that !